src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -127,13 +127,21 @@
             "group size would be approximately 10 reads.")
     public int MAX_GROUP_RATIO = 500;
 
+    @Option(doc = "Barcode SAM tag (ex. BC)", optional=true)
```

Comment:
Looking at your NA12878 Bam file, the read group is empty, so using the ```BC``` tag in addition to ```BX/RX``` seems reasonable.  What do you think @wheaton5? 

Also, what should we do for reads that are missing an ```RX``` or ```BX``` tag?   In the currently implementation, they would all be compared against each other, which is conservative.